index.html

```diff
@@ -168,6 +168,24 @@ <h3 class="language-title">Skills</h3>
                 </div>
             </div>
         </section>
+        <section class="section-contuct-me">
+            <div class="container">
+                <h2 class="contuct-me__heading">Contact me</h2>
+                <p class="contuct-me__paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
+                <input type="text" class="contuct-me__name-input" placeholder="Enter Your Name">
+                <input type="email" class="contuct-me__email-input" placeholder="Enter Your Email">
+                <textarea name="Enter Your massages Here" id="" cols="30" rows="10" class="contuct-me__textarea" placeholder="Enter your massages here"></textarea>
+                <a href="#" class="contuct-me__btn">Get in touch</a>
+            </div>
+        </section>
+
+        <!-- <section class="section-footer">
+            <nav>
+                <ul>
+                    <li><a href="#">home</a></li>
+                </ul>
+            </nav>
+        </section> -->
```

Comment:
please don't leave unused code in your file 😄 .

Unused code left as comments makes it hard to read your code and can be a source of bugs in the future 👍 .